src/frontends/paddle/src/op/scatter.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+#include "openvino/opsets/opset15.hpp"
+#include "openvino/opsets/opset4.hpp"
```

Comment:
opset15 is back compatible to previous opsets. We can keep only opset15 header here.